docs/upgrade.adoc

```diff
@@ -7,6 +7,10 @@ JanusGraph release.
 
 === Upgrading to JanusGraph 0.4.0
 
+The version of HBase that is included in the distribution of JanusGraph was upgraded from 1.2.6 to 2.1.5.
```

Comment:
(nitpick) Please add a headline like:
```
==== HBase: Upgrade from 1.2 to 2.1
```

All upgrade docs for this version are backend specific and users probably only want to read the aspects for their backends.

Reply:
Done.